Module-3 || DBMS Schema Design/DBSchema.txt

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE projects
+(
+  ProjectID INT UNSIGNED,
```

Comment:
property names are lowercase and separated by '_' . Please rename the properties accordingly.